docs/faq.md

```diff
@@ -16,3 +16,9 @@ We list some potential troubles encountered by users and developers, along with
   pip uninstall pycocotools mmpycocotools
   pip install mmpycocotools
   ```
+
+- If you face the error shown below when your numba == 0.48.0 and numpy >= 1.20.0
```

Comment:
If you face the error shown below, and your environment contains numba == 0.48.0 with numpy >= 1.20.0